cypress/tsconfig.json

```diff
@@ -0,0 +1,32 @@
+{
+  "compilerOptions": {
+    "baseUrl": ".",
+
+    "target": "es5",
+    "lib": ["es5", "dom", "es2015"],
```

Comment:
Any reason in specific to keep the target this old?

Bringing it up to the same level as used in `client/tsconfig.json` would be:
```json
    "module": "es2022",
    "target": "es2022",
    "lib": ["ES2022", "DOM", "DOM.Iterable"],
```